src/modules/fancyzones/editor/FancyZonesEditor/GridZone.xaml.cs

```diff
@@ -112,6 +112,7 @@ private void GridZone_SizeChanged(object sender, SizeChangedEventArgs e)
             // using current culture as this is end user facing
             WidthLabel.Text = Math.Round(ActualWidth).ToString(CultureInfo.CurrentCulture);
             HeightLabel.Text = Math.Round(ActualHeight).ToString(CultureInfo.CurrentCulture);
+            System.Windows.Automation.AutomationProperties.SetName(this, FancyZonesEditor.Properties.Resources.Zone_Name + " " + (_zone.Index + 1).ToString(CultureInfo.CurrentCulture) + ". " + WidthLabel.Text + " x " + HeightLabel.Text);
```

Comment:
instead of `number X number`, maybe to announce `width: number, height: number` ?

Reply:
Thank you for the review. Addressed it in the latest commit. PTAL